src/cbc_sdk/platform/alerts.py

```diff
@@ -84,47 +84,48 @@ class Alert(PlatformModel):
+        "alert_notes_present": "notes_present",
         "backend_timestamp": "create_time",
-        "first_event_timestamp": "first_event_time",
-        "last_event_timestamp": "last_event_time",
         "backend_update_timestamp": "last_update_time",
-        "k8s_namespace": "namespace",
-        "alert_notes_present": "notes_present",
-        "device_policy_id": "policy_id",
+        "determination_value": "alert_classification.user_feedback",
         "device_policy": "policy_name",
+        "device_policy_id": "policy_id",
+        "device_target_value": "target_value",
+        "first_event_timestamp": "first_event_time",
+        "k8s_cluster": "cluster_name",
+        "k8s_kind": "workload_kind",
+        "k8s_namespace": "namespace",
+        "k8s_pod_name": "replica_id",
+        "k8s_workload_name": "workload_name",
+        "last_event_timestamp": "last_event_time",
+        "ml_classification_final_verdict": "alert_classification.classification",
+        "ml_classification_global_prevalence": "alert_classification.global_prevalence",
+        "ml_classification_org_prevalence": "alert_classification.org_prevalence",
         "netconn_local_port": "port",
         "netconn_protocol": "protocol",
         "netconn_remote_domain": "remote_domain",
         "netconn_remote_ip": "remote_ip",
+        "parent_guid": "threat_cause_parent_guid",
+        "primary_event_id": "created_by_event_id",
+        "process_guid": "threat_cause_process_guid",
+        "process_issuer": "threat_cause_actor_certificate_authority",
+        "process_md5": "threat_cause_actor_md5",
+        "process_name": "threat_cause_actor_name",
+        "process_publisher": "threat_cause_actor_publisher",
+        "process_reputation": "threat_cause_reputation",
+        "process_sha256": "threat_cause_actor_sha256",
+        "remote_k8s_kind": "remote_workload_kind",
         "remote_k8s_namespace": "remote_namespace",
         "remote_k8s_pod_name": "remote_replica_id",
-        "remote_k8s_kind": "remote_workload_kind",
         "remote_k8s_workload_name": "remote_workload_name",
-        "k8s_pod_name": "replica_id",
         "rule_id ": "rule_id",
         "run_state": "run_state",
-        "device_target_value": "target_value",
-        "process_issuer": "threat_cause_actor_certificate_authority",
-        "process_name": "threat_cause_actor_name",
-        "process_publisher": "threat_cause_actor_publisher",
-        "process_sha256": "threat_cause_actor_sha256",
-        "primary_event_id": "threat_cause_cause_event_id",
-        "process_md5": "threat_cause_md5",
-        "parent_guid": "threat_cause_parent_guid",
-        "process_reputation": "threat_cause_reputation",
         "ttps": "threat_indicators",
         "watchlists.id": "watchlists",
         "workflow.change_timestamp": "workflow.last_update_time",
-        "workflow.note": "workflow.comment",
         "workflow.closure_reason": "workflow.remediation",
```

Comment:
Remove mapping 
"workflow.closure_reason": "workflow.remediation",
Received new info - remediation is free text, closure_reason is enum, they do not have the same value.

Reply:
Further work will be required for workflow, tracked on CBAPI-4807.
For this PR, remove the mapping between closure_reason and remediation.